index.js

```diff
@@ -0,0 +1,13 @@
+function calculation(items) {
```

Comment:
The name "doesn't matter" from a purely technical perspective. Whatever name you put here is what the value passed will be called. But from a readability perspective it should be `item`. 

You have an array called `items` which is a list of objects. Map will loop through that array and will call `calculation` passing in one of those objects. Since the list is `items` a single object from the list would be an `item`.

If the `items` array contains 3 objects, then the `calculation` function will be called 3 times. Each call will receive one of the objects.